src/crystal/system/fiber.cr

```diff
@@ -0,0 +1,16 @@
+module Crystal::System::Fiber
+  # Allocates memory for a stack.
+  # def self.allocate_stack(stack_size : Int) : Void*
+
+  # Frees memory of a stack.
+  # def self.free_stack(stack : Void*, stack_size : Int) : Nil
+
+  # Determines location of the top of the main fiber's stack.
```

Comment:
This is actually the main *process* stack. Happens to also be the main fiber stack.